pom.xml

```diff
@@ -264,8 +264,7 @@
     <version.org.sonatype.sisu>2.3.0</version.org.sonatype.sisu>
     <version.org.sonatype.sisu.sisu-guice>3.2.3</version.org.sonatype.sisu.sisu-guice>
     <version.org.springframework>5.2.9.RELEASE</version.org.springframework>
-    <version.org.springframework.boot>2.3.4.RELEASE</version.org.springframework.boot>
-    <version.org.springframework.amqpboot>2.3.4</version.org.springframework.amqpboot>
+    <version.org.springframework.boot>2.4.9</version.org.springframework.boot>    
```

Comment:
Shouldn't we use 2.5.x or 2.6.x?

Reply:
ok, i can see the jira now is about to certify SB 2.4.x. so ignore my previous comment